src/hardware/ipx.cpp

```diff
@@ -1045,10 +1045,18 @@ class IPXNET final : public Program {
 				WriteOut("Sending broadcast ping:\n\n");
 				pingSend();
 				ticks = GetTicks();
-				while((GetTicks() - ticks) < 1500) {
+				while ((GetTicksSince(ticks)) < 1500) {
 					CALLBACK_Idle();
 					if(pingCheck(&pingHead)) {
-						WriteOut("Response from %d.%d.%d.%d, port %d time=%dms\n", CONVIP(pingHead.src.addr.byIP.host), SDLNet_Read16(&pingHead.src.addr.byIP.port), GetTicks() - ticks);
+						WriteOut(
+						        "Response from %d.%d.%d.%d, port %d time=%dms\n",
+						        CONVIP(pingHead.src.addr
+						                       .byIP.host),
+						        SDLNet_Read16(
+						                &pingHead.src
+						                         .addr
+						                         .byIP.port),
```

Comment:
Maybe unwrap just the `CONVIP` and `SDLNet_Read16`; even if they go beyond 80 a bit, it will probably be a bit easier to read.

``` c++
WriteOut("Response ... ",
         CONVIP( ... ),
         SDLNet_Read16( ... ),
         GetTicksSince(ticks));
 ```